### ISSUE TITLE
The launcher should not allow unpausing

### ISSUE THREAD

**Issue by uecasm** [DFHack/dfhack#3125]
Pressing the spacebar while typing a command in `gui/launcher` currently (as of 50.07-beta1 on Steam) unpauses/repauses.  Since many DF commands require spaces, and the launcher is mostly-full-screen and there's not really any good reason to interact with the map while it's open (and many good reasons not to), this is bad.

**Comment by myk002** [comment]
Yes, this is a feature, but a bug, since otherwise there is not a way to pause the game with the keyboard when you bring up gui/launcher and you have autopausing turned off in gui/control-panel.

Documented here! https://docs.dfhack.org/en/latest/docs/tools/gui/launcher.html#editing-and-running-commands

**Comment by Cubittus** [comment]
How about:
![Dwarf_Fortress-2023-03-30-150227-050](https://user-images.githubusercontent.com/627213/228862124-03880acf-63a2-4583-99be-f44eb9ca8310.png)

**Comment by uecasm** [comment]
Ok, it appears I must have been double-tapping space.  I don't remember doing that yesterday, but I can't get it to happen today without doing that.

Perhaps the hotkeys at the bottom could show `space space: unpause`?  Though perhaps only if the DF hotkey hasn't been changed.
